README.md

```diff
@@ -120,62 +123,59 @@ Will configure the varnish repo specified by `node['varnish']['major_version']`
 
 #### varnish_repo - Properties
 
-| Name            | Type                                                   | Default Value                                                              |
```

Comment:
This should be split out in the documentation folder for easier reading and linked to.